misc/ansible/screenly.yml

```diff
@@ -0,0 +1,38 @@
+- hosts: all
+  user: pi
+  vars:
+    screenly_systemd_units:
+      - X.service
+      - matchbox.service
+      - screenly-viewer.service
+      - screenly-web.service
+
+  tasks:
+    - name: Ensure screenly config dir exists
+      file: path=/home/pi/.screenly state=directory
+
+    - name: Copy screenly default config
+      copy: src=/home/pi/screenly/misc/screenly.conf dest=/home/pi/.screenly/screenly.conf
```

Comment:
Hm, I forgot to set force=no. According to doc http://docs.ansible.com/ansible/copy_module.html